docs/source/posts/2024/2024-06-15-week2-wachiou-bouraima.rst

```diff
@@ -0,0 +1,52 @@
+WEEK 2: Refinements and Further Enhancements
+============================================
+
+.. post:: June 15, 2024
+   :author: Wachiou BOURAIMA
+   :tags: google
+   :category: gsoc
+
+Hello again,
+~~~~~~~~~~~~~
+
+Welcome back to my Google Summer of Code (GSoC) 2024 journey! This week has been dedicated to refining and improving the work done so far, with a particular focus on the keyword_only decorator.
+
+
+Renaming and Updating the Decorator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This week, I renamed the keyword_only decorator to warn_on_args_to_kwargs for greater clarity. The updated decorator now includes version parameters from_version and until_version. This enhancement ensures that the decorator will raise a RuntimeError if the current version of FURY is greater than until_version.
```

Comment:
Using text formatting will be good when mentioning code snippets.

ex:
warn_on_args_to_kwargs -> `warn_on_args_to_kwargs`

Reply:
Hi @robinroy03

Thank you for yout feedback. Okay